effekt/jvm/src/test/scala/effekt/ChezSchemeTests.scala

```diff
@@ -37,6 +37,9 @@ abstract class ChezSchemeTests extends EffektTests {
     examplesDir / "pos" / "unsafe_cont.effekt",
     examplesDir / "pos" / "propagators.effekt",
 
+    // bytearray is not implemented
+    examplesDir / "pos" / "string_interpolation.effekt",
+
```

Comment:
With #791 merged, you should be able to remove this after rebasing on `master` :)
```suggestion
```